puppet-module-repose.spec

```diff
@@ -32,6 +32,8 @@ cp -pr * %{buildroot}%{module_dir}/
 %changelog
 * Fri Nov 10 2017 Josh Bell <josh.bell@rackspace.com> - 2.4.0-1
 - Add support for Repose 8 tracing header options
+* Tue Oct 31 2017 Senthil Natarajan <senthil.natarajan@rackspace.com> - 2.3.0-1
```

Comment:
This seems out of date.

Reply:
yeah, this is from the initial commit, a while back. i can pull it out.